services/swagger.js

```diff
@@ -134,31 +135,37 @@ exports.addFormat = function (format, validationFunction) {
     tv4.addFormat(format, validationFunction);
 };
 
-function overrideInheritedProperties(definitions) {
+function flattenModelDefinitions(definitions) {
     Object.keys(definitions).forEach(function (defn) {
-        var props = {};
+        var flattenedDefn = {};
```

Comment:
I know it'll get hoisted up to here, but for readibility, move the variable declaration inside the `if` statement